### PR TITLE
Airlines index

### PR DESCRIPTION
**Pull request by wanjoc**
Completed this as a group

### CODE REVIEW

File: src/main/frontend/client/components/AirlineTile.js [hunk 1]
```diff
@@ -0,0 +1,8 @@
+import React from "react"
+
+const AirlineTile = props => {
+  const { name } = props.airline
```
Comment (safang99) [inline src/main/frontend/client/components/AirlineTile.js:4]: You could even refactor this to use nested destructuring on line 3.  
For example:

```js
const AirLineTile = ({airplane: {name}}) => {
```

File: src/main/java/com/launchacademy/reviews/models/Airline.java [hunk 2]
```diff
@@ -0,0 +1,50 @@
...
+
+  @NotBlank
+  @Column(name = "name", nullable = false)
+  private String name;
+
+  @NotBlank
+  @Column(name="description", nullable = false)
+  private String description;
+
+  @NotBlank
```
Comment (safang99) [inline src/main/java/com/launchacademy/reviews/models/Airline.java:35]: I noticed that this field is optional based on the migration, but `@NotBlank` has been added and there are no entries that don't omit this field in the seeder.  I would drop the db, recreate it and then run the seeder with a new entry added that doesn't include a `logo_url`

File: src/main/java/com/launchacademy/reviews/seeders/AirlineSeeder.java [hunk 3]
```diff
@@ -0,0 +1,48 @@
...
+      airlineTwo.setLogoUrl("https://www.newrest.eu/wp-content/uploads/2016/01/United-Airlines.jpg");
+      airlineService.save(airlineTwo);
+
+      Airline airlineThree = new Airline();
+      airlineThree.setName("Delta Airlines");
+      airlineThree.setDescription("Delta Air Lines, Inc., typically referred to as Delta, is one of the major airlines of the United States and a legacy carrier. One of the world's oldest airlines in operation, Delta is headquartered in Atlanta, Georgia.");
+      airlineThree.setHeadquarters("Atlanta, GA");
+      airlineThree.setContactNumber(" 1 (800) 221-1212");
+      airlineThree.setHomepageUrl("https://www.delta.com/");
+      airlineThree.setLogoUrl("https://mma.prnewswire.com/media/588812/DELTA_AIR_LINES_LOGO.jpg?p=facebook");
```
Comment (safang99) [inline src/main/java/com/launchacademy/reviews/seeders/AirlineSeeder.java:44]: I would either add a 4th Airline that doesn't include a logo or remove line 44 to test if it is currently allowing null

**Review by safang99 — CHANGES_REQUESTED** [review]
Amazing work on the first PR!! I left comments on one potential refactor and one issue with an optional field that I just want to ensure was tested before merging.  If you had already checked that just let me know and I can go ahead to merge this 👍

File: src/main/java/com/launchacademy/reviews/seeders/AirlineSeeder.java (hunk 3 above)
Comment (wanjoc) [reply src/main/java/com/launchacademy/reviews/seeders/AirlineSeeder.java:44]: We tested out the 3rd airline without logoUrl and worked. Removed @NotBlank on logo_url
